spec/features/visitors/sign_up_spec.rb

```diff
@@ -303,7 +303,7 @@
   #   When I resend confirmation instructions to an existing user
   #   Then the user does not receive an email
   context 'confirmation instructions sent to existing user', email: true do
-    it 'does not send an email to the existing user' do
+    xit 'does not send an email to the existing user' do
```

Comment:
this 'resend email confirmation' link is not currently in designs, hence the removal -- will discuss with ryan t and figure out where we can slide it...